deploy.config

```diff
@@ -0,0 +1,8 @@
+[deploy]
+region: ca-central-1
+stack_name: reinvent-2018
```

Comment:
Any objection to either keeping this called something like `sid345` (which will get caught in any updates that we need to make anyway if the designation changes), or at least something more specific than `reinvent-2018`?

(Ditto for the below.)